src/shims/unix/linux/epoll.rs

```diff
@@ -166,25 +376,160 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
     fn epoll_wait(
         &mut self,
         epfd: &OpTy<'tcx>,
-        events: &OpTy<'tcx>,
+        events_op: &OpTy<'tcx>,
         maxevents: &OpTy<'tcx>,
         timeout: &OpTy<'tcx>,
-    ) -> InterpResult<'tcx, Scalar> {
+        place: &'tcx MPlaceTy<'tcx>,
```

Comment:
If this is the `dest`  place, please call it `dest` so that the reader has some chance of understanding the purpose of the place.